packages/components/status/stories/status.stories.js

```diff
@@ -0,0 +1,24 @@
+import { storiesOf, html, select } from '@open-wc/demoing-storybook';
+import { withKnobs, text } from '@storybook/addon-knobs';
+import { helpers } from '@tradeshift/elements';
+
+import '@tradeshift/elements.status';
+import { STATUS_TYPE } from '../src/utils/constants';
+
+storiesOf('ts-status', module)
+	.addDecorator(withKnobs)
+	.add('default', () => {
+		const status = select(
+			'Status',
+			{
+				default: '',
+				...helpers.objectKeysChangeCase(STATUS_TYPE)
+			},
+			''
+		);
+		const statusText = text('text', 'Ok Danger Note');
+
+		return html`
+			<ts-status status="${status}" ?text="${statusText}"> </ts-status>
```

Comment:
Please, remove the `?` from the non-boolean attributes.

Reply:
done